tests/examples/pacer/dockets/district/dcd_3.html

```diff
@@ -0,0 +1,136 @@
+Cause: 15:1 Antitrust Litigation</td>
+<td width="40%" valign="top"><br>Date Filed: 10/20/2020<br>
+Jury Demand: None<br>
+Nature of Suit: 410 Anti-Trust<br>
+Jurisdiction: U.S. Government Plaintiff</td>
+</tr></tbody></table>
+
+	<form enctype="multipart/form-data" method="post" name="view_multi_docs" action="show_multidocs.pl?caseid=223205&amp;pdf_header=2&amp;pdf_toggle_possible=0"><input type="hidden" name="caseid" value="223205"><input type="hidden" name="docket_caseid" value="223205"><input type="hidden" name="download_documents" value=""><input type="hidden" name="sort1" value="oldest date first"><input type="hidden" name="date_range_type" value="Filed"><br><table rules="all" width="99%" cellspacing="0" cellpadding="5" border="1" align="center">
+<tbody><tr><td style="font-weight:bold; width=94; white-space:nowrap">Date Filed</td>
+<th>#</th><td style="font-weight:bold; white-space:nowrap"><small>Select<br><a href="#" onclick="select_all_documents(true, false)" '="">all</a> / <a href="#" onclick="select_all_documents(false, false)" '="">clear</a></small></td><td style="font-weight:bold">Docket Text</td></tr>
+<tr><td width="94" valign="top" nowrap="">10/20/2020</td><td style="white-space:nowrap" valign="top" align="right"><a href="https://ecf.dcd.uscourts.gov/doc1/04508117526" onclick="goDLS('/doc1/04508117526','223205','15','','2','','','','');return(false);">1</a>&nbsp;</td><td valign="top" align="center"><input type="checkbox" name="all_documents" value="223205_15" onclick="select_all_attachments(this);"><br><div class="show_filesize">  1.2 MB</div></td>
+<td valign="top"><!--SB-->COMPLAINT against GOOGLE LLC filed by UNITED STATES OF AMERICA. (Attachments: # <a href="https://ecf.dcd.uscourts.gov/doc1/04518117527" onclick="goDLS('/doc1/04518117527','223205','15','','2','','','','');return(false);">1</a> Civil Cover Sheet, # <a href="https://ecf.dcd.uscourts.gov/doc1/04518117528" onclick="goDLS('/doc1/04518117528','223205','15','','2','','','','');return(false);">2</a> Summons)(ztnr) (Entered: 10/20/2020)</td></tr>
+<tr><td></td><td></td><td></td><td><table style="margin:10px;width:500px"> <tbody><tr>
+			<td style="width:20px"><input name="document_223205_15_0" type="checkbox" value="8117526-1074119-64" autocomplete="off" onchange="update_sizes(this)"></td>
+			<td>0</td><td>Main Document</td>
+			<td>64 pages</td>
+			<td>  1.0 MB</td>
+		</tr>  <tr>
+			<td style="width:20px"><input name="document_223205_15_1" type="checkbox" value="8117527-38495-2" autocomplete="off" onchange="update_sizes(this)"></td>
+			<td>1</td><td>Civil Cover Sheet</td>
+			<td>2 pages</td>
+			<td>37.6 KB</td>
+		</tr>  <tr>
+			<td style="width:20px"><input name="document_223205_15_2" type="checkbox" value="8117528-75539-2" autocomplete="off" onchange="update_sizes(this)"></td>
+			<td>2</td><td>Summons</td>
+			<td>2 pages</td>
+			<td>73.8 KB</td>
+		</tr></tbody></table></td></tr>
```

Comment:
@albertisfu Would you happen to have any more samples of these dockets with attachments? Would be nice to have some more to test against for #718.

Reply:
Just FYI, I found some examples. :)